plugins/module_utils/job.py

```diff
@@ -525,7 +526,44 @@ def _ddname_pattern(contents, resolve_dependencies):
         re.IGNORECASE,
     ):
         raise ValueError(
-            'Invalid argument type for "{0}". expected "ddname_pattern"'.format(
+            'Invalid argument type for "{0}". Expected "ddname_pattern"'.format(
+                contents
+            )
+        )
+    return str(contents)
+
+
+def _username_pattern(contents, resolve_dependencies):
```

Comment:
other type validations are in `plugins/module_utils/better_arg_parser.py` I think this should be included there too.